verification.py

```diff
@@ -669,8 +671,88 @@ def _run_mutation_discoal_stats(self, args):
         )
         self._plot_stats("mutation", df_d, df_msp, "discoal", "msp")
 
+    def _discoal_str_to_simulation(self, args):
+        # takes discoal command line as input
+        # and returns msprime run treeseqs
 
-class DisccoalCompatibility(DiscoalTest):
+        tokens = args.split(" ")
+        # positional args
+        sample_size = int(tokens[0])
+        nreps = int(tokens[1])
+        seq_length = int(tokens[2])
+        # parse discoal command line for params
+        # init ones we definitely need for comparison
+        theta = rho = alpha = sweep_site = sweep_mod_time = None
+        refsize = 1e6
+        for i in range(3, len(tokens)):
+            # pop size change case
+            if tokens[i] == "-en":
+                logging.error(
```

Comment:
I think these should be exceptions - just ``raise ValueError(...)`` instead of calling logging.error and returning.

Reply:
> I think these should be exceptions - just `raise ValueError(...)` instead of calling logging.error and returning.

I think this might be change request before merging the PR. @andrewkern